wanikani-show-context-sentence.user.js

```diff
@@ -189,16 +189,16 @@ const quizInput = document.querySelector('.quiz-input');
   if (quizInput) {
     quizInput.insertAdjacentHTML('beforebegin', `
       <div class="wf-sawarabimincho">
-        <span>${sentence}</span>
-        <span style="background-color: white">${translation}</span>
+        <span data-type="sentence">${sentence}</span>
+        <span data-type="translation" style="background-color: white">${translation}</span>
```

Comment:
Did you consider making the text color purple, and then unmasking by setting the background color to white on hover over?

That way you wouldn't have a distracting large white box during the review.